web/ui/dashboard/src/app/private/pages/project/apps/apps.component.ts

```diff
@@ -79,4 +90,11 @@ export class AppsComponent implements OnInit {
 			return error;
 		}
 	}
+
+	deleteApplication(event: MouseEvent, app: APP) {
```

Comment:
I think `promptDeleteModal` will be a better name here. It better describes the action to be taken here.